src/test/java/com/laserfiche/api/client/integration/UsernamePasswordHandlerTest.java

```diff
@@ -0,0 +1,162 @@
+package com.laserfiche.api.client.integration;
+
+import com.laserfiche.api.client.httphandlers.*;
+import com.laserfiche.api.client.model.ApiException;
+import kong.unirest.HttpStatus;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+@Tag("APIServer")
```

Comment:
So we have `Tag("Cloud")`, would something like `Tag("SelfHosted")` make more sense than `@Tag("APIServer")`?

Reply:
done